conf/app.routes

```diff
@@ -31,6 +31,9 @@ GET         /:taxYear/pensions-summary
 
 GET         /:taxYear/overseas-pensions                                                                                            controllers.pensions.OverseasPensionsSummaryController.show(taxYear: Int)
 
+GET         /:taxYear/:journey/section-completed                                                                                   controllers.pensions.SectionCompletedStateController.show(taxYear: Int, journey: String)
```

Comment:
Was the thinking to have `controllers...(..., journey: **Journey**)`? I assume this was the intent hence the `PathBindable`?

Reply:
Yep, changed it and forgot to change it back! Thanks